### DISCUSSION TITLE
Bluetooth!

### DISCUSSION THREAD

**Discussion by cranstantinople** [microsoft/WSA#420] [General]
I bought a Galaxy Watch 6 Classic before realizing I couldn't pair it with iOS like I could my previous Samsung Gear Watches.  I was willing to lose all the features of it being paired to my phone and was able to install the Galaxy Wearable's app on Windows 11 but then realized it couldn't access Bluetooth.  I can't/don't want to move to a Galaxy Phone but the Galaxy Watch Classic with Rotating Bezel is overall a great looking watch.  Sad that everyone seems to be moving away from cross-platform compatibility.  Bluetooth for Android Subsytem would be very helpful

**Comment by jake-brown-maps** [comment]
Logged as a known issue: https://github.com/microsoft/WSA/issues/90

**Reply by cranstantinople** [reply]
Sorry about that.  I searched "blue" thinking it would show up but I think it only matches the full word-- still kinda new to Github.  I found the existing issue now.  Thanks!
